marketing/src/index.css

```
@@ -0,0 +1,333 @@
+@import "tailwindcss";
+
+/* 👇 INI KUNCI UTAMANYA: Beritahu Tailwind untuk scan folder web */
+@source "./src";
```

Stylelint (17.4.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)